pull-request.sh

```diff
@@ -49,6 +50,7 @@ create_pull_request() {
     TARGET="$(echo -n "${2}" | jq --raw-input --slurp ".")"  # pull request TO this target
     BODY="$(echo -n "${3}" | jq --raw-input --slurp ".")"    # this is the content of the message
     TITLE="$(echo -n "${4}" | jq --raw-input --slurp ".")"   # pull request title
+    MAINTAINER_CAN_MODIFY="$(echo -n "${5}" | jq --raw-input --slurp ".")"   # pull request title
 
```

Comment:
The comment here for what the variable means needs to be updated (it still mentions the pull request title)

Reply:
whops :) right! removing it ✍️